boards/snps/nsim/arc_v/rmx5xx.dtsi

```diff
@@ -0,0 +1,50 @@
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+			timebase-frequency = <5000000>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			cpu0: cpu@0 {
+				compatible = "snps,av5rmx", "riscv";
+				device_type = "cpu";
+				reg = <0>;
+				clock-frequency = <5000000>;
+				riscv,isa = "rv64imafdc";
+				#cooling-cells = <2>;
+
+				cpu0_intc: interrupt-controller {
+					compatible = "riscv,cpu-intc";
+					interrupt-controller;
+					#address-cells = <0>;
+					#interrupt-cells = <1>;
+				};
```

Comment:
I'm wondering what is this interrupt controller. I'd expect that CLINT should be the root interrupt controller.

Reply:
I'm not pretty sure about this part. It looks like interface for CLINT. And CLINT has reference to cpu0_intc